lib/content-services/src/lib/aspect-list/aspect-list.component.ts

```diff
@@ -57,13 +57,9 @@ export class AspectListComponent implements OnInit, OnDestroy {
     ngOnInit(): void {
         if (this.nodeId) {
             const node$ = this.nodeApiService.getNode(this.nodeId);
-            const customAspect$ = this.aspectListService.getCustomAspects()
-            .pipe(map(
```

Comment:
Why have you removed this part? the aspect needs to be filtered when showed up and only the one in the config should be showed.